From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ram Muthiah <rammuthiah@google.com>
Date: Fri, 9 Aug 2019 12:48:12 -0700
Subject: ANDROID: Removed unnecessary modules from cuttlefish.

Modules do not succesfuly load on cuttlefish and are not needed for gki.

Bug: 132629930
Test: Treehugger + Launch local KO Ramdisk
Signed-of-by: Ram Muthiah <rammuthiah@google.com>
Change-Id: I07b6c140e4c474d66132a4fb5630d18933b69e81
---
 arch/x86/configs/gki_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 731cec5e1bc3..2d5f55d7b966 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -242,11 +242,13 @@ CONFIG_HW_RANDOM_VIRTIO=y
 CONFIG_GPIOLIB=y
 # CONFIG_HWMON is not set
 CONFIG_DEVFREQ_THERMAL=y
+# CONFIG_X86_PKG_TEMP_THERMAL is not set
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_DRM=y
 # CONFIG_DRM_FBDEV_EMULATION is not set
 CONFIG_DRM_VIRTIO_GPU=y
+# CONFIG_LCD_CLASS_DEVICE is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_SOUND=y
 CONFIG_SND=y
